src/content/community/conferences.md

```diff
@@ -25,6 +25,12 @@ May 27 - 31, 2025. In-person in Athens, Greece
 
 [Website](https://athens.cityjsconf.org/) - [Twitter](https://x.com/cityjsconf) - [Bluesky](https://bsky.app/profile/cityjsconf.bsky.social)
 
+### React Norway 2025 {/*react-norway-2025*/}
+June 13, 2025. In-person in Oslo, Norway + remote (virtual event)
+
+[Website](https://reactnorway.com/) - [Twitter](https://x.com/ReactNorway)
+
+
```

Comment:
super nit but could you remove this extra newline

Reply:
Entirely right, apologies for the mistake 😊